br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -94,6 +95,7 @@ const (
 			raw_value   mediumblob NOT NULL COMMENT 'the value of the conflicted key',
 			raw_handle  mediumblob NOT NULL COMMENT 'the data handle derived from the conflicted key or value',
 			raw_row     mediumblob NOT NULL COMMENT 'the data retrieved from the handle',
+			is_data_kv  tinyint(1) NOT NULL,
```

Comment:
maybe `kv_group`, values can be `data`/`index`, or if we want finer grain, we can use index id as the name.

current is lgtm too

Reply:
maybe current `is_data_kv` is simpler in implemention, it seems like it is not necessary to determine whether the value is `data` or `index` based on the result `tablecodec.IsRecordKey(conflictInfo.RawKey)`